xfr_test.go

```diff
@@ -0,0 +1,195 @@
+package dns
+
+import (
+	"fmt"
+	"net"
+	"sync"
+	"testing"
+	"time"
+)
+
+var (
+	tsigSecret = map[string]string{"axfr.": "so6ZGir4GPAqINNh9U5c3A=="}
+	xfrSoa     = `miek.nl.	0	IN	SOA	linode.atoom.net. miek.miek.nl. 2009032802 21600 7200 604800 3600`
+	xfrA       = `x.miek.nl.	1792	IN	A	10.0.0.1`
+	xfrMX      = `miek.nl.	1800	IN	MX	1	x.miek.nl.`
+	testData   = axfrZoneData()
```

Comment:
I don’t think this needs to be a seperate function and should be renamed. `xfrTestData = []RR{...}` and then `xfrSoa = testRR("...")`.